addons/website/static/src/js/menu/seo.js

```diff
@@ -613,10 +614,25 @@ var SeoConfigurator = Dialog.extend({
             self.renderPreview();
         });
     },
-    descriptionChanged: function () {
+    descriptionChanged: function (ev) {
         var self = this;
         _.defer(function () {
-            var description = self.$('textarea[name=seo_page_description]').val();
+            var $textarea = $(ev.target);
+            var description = $textarea.val();
+            var $warning = $textarea.next('div#website_meta_description_warning');
+            var length = $textarea.val().length;
+            if (length > self.maxRecommendedDescriptionSize || length < self.minRecommendedDescriptionSize) {
```

Comment:
As 0 length description is acceptable, can you add check condition if not length ?
The preview is corretly pointing the fact an empty description will be handled and set automatically

Reply:
Ok